util/gconv/gconv_struct.go

```diff
@@ -377,6 +377,12 @@ func bindVarToStructAttr(structReflectValue reflect.Value, attrName string, valu
 	if empty.IsNil(value) {
 		structFieldValue.Set(reflect.Zero(structFieldValue.Type()))
 	} else {
+		// Try to call custom converter.
+		// Issue: https://github.com/gogf/gf/issues/3099
+		if ok, err := callCustomConverter(reflect.ValueOf(value), structFieldValue); ok {
```

Comment:
It'd be better to check that the `value` is already type of `reflect.Value` or not.

Reply:
improved